### DISCUSSION TITLE
Can't get verified for student status

### DISCUSSION THREAD

**Discussion by Andreas-Mollberg** [community/community#73024] [GitHub Education]
### Hi everyone,

My school does not use Student ID's or school email domains but I have submitted a copy of my acceptance letter with the school letterhead on it, but I still get the message that I need to upload a picture of my school ID.

**Comment by InvisiblePro** [comment]
Your school acceptance letterhead can be accepted by GitHub, it requires having a `date` i.e. `10-Oct-20xx` mentioned on the letterhead. It will take time of at least 1 week time after you submit your documents/proofs. Even though I didn't have my school ID, I have been approved as a Student because it requires proof with a `date`. And also, you need to update your [GitHub billing address](https://github.com/settings/billing/payment_information).
Have a good journey ✨
If this worked, please make sure to "mark the answer".
